### ISSUE TITLE
Markdown Preview Error when Pandoc Selected as Parser

### ISSUE THREAD

**Issue by pa-0** [RickStrahl/MarkdownMonster#1133]
# Pandoc Parser Error in Preview Pane

## Expected Behavior

Markdown is properly displayed in Preview Pane

## Actual Result

![image](https://github.com/user-attachments/assets/8b59d6f4-a502-4f36-ae03-b2ca85a70a9c)

## Additional Notes

<details><summary><h3>Steps taken to troubleshoot:</h3></summary>

1. Reinstalled `pandox`
2. Reinstalled `pandoc` parser adddon for Markdown Monster
3. Restarted app / machine
</details>
<details><summary><h3>Error Message</h3></summary>
<em>(Copied & pasted directly from MM's preview pane)</em>

## Unable to render Markdown Document
 An error occurred trying to parse the Markdown document to HTML.
 
#### Method not found: 'System.String MarkdownMonster.MarkdownParserBase.ParseFontAwesomeIcons(System.String)'.
[more info...](https://github.com/RickStrahl/MarkdownMonster/issues/new#0)
 
#### Error Detail
```
at PanDocMarkdownParserAddin.PandocMarkdownParser.Parse(String markdown)
at MarkdownMonster.MarkdownDocument.RenderHtml(String markdown, Boolean usePragmaLines, Boolean noBanner,
IMarkdownParser parser) in D:\projects\MarkdownMonsterCode\markdownmonster\_Classes\MarkdownDocument.cs:line 1299
```

</details>

**Comment by RickStrahl** [comment]
Should be fixed in v0.5 of the Pandoc Addin. You need to update the addin from the Addin Manager.

**Comment by pa-0** [comment]
No - thank YOU for the quick turnaround on a fix.
